apps/shell/tash_main.c

```diff
@@ -49,10 +49,118 @@ enum tash_input_state_e {
 #define TASH_TASK_STACKSIZE   (4096)
 #define TASH_TASK_PRIORITY    (125)
 
+#define TASH_MAX_STORE        (10)
```

Comment:
This had better defined in kconfig, I think.

Reply:
done